CactbotOverlay/CactbotOverlay.csproj

```diff
@@ -63,13 +63,11 @@
     <Reference Include="Newtonsoft.Json">
       <HintPath>ThirdParty\OverlayPlugin\Newtonsoft.Json.dll</HintPath>
     </Reference>
-    <Reference Include="OverlayPlugin">
```

Comment:
Can you update [the installation text](https://github.com/quisquous/cactbot/blob/master/CactbotOverlay/ThirdParty/OverlayPlugin/dummy.txt#L11) accordingly?